kolibri/core/logger/api.py

```diff
@@ -611,7 +611,8 @@ def _update_and_return_mastery_log_id(
                     mastery_level=context["mastery_level"],
                     summarylog_id=summarylog_id,
                 )
-                update_fields = tuple()
+                update_fields = ("end_timestamp", "_morango_dirty_bit")
```

Comment:
wondering if we could win some performance doing this inside the if time_spent_delta: condition below. If there has not been any difference of time we don't need to update it nor force morango to sync the dates

Reply:
Yes, that does make sense, and preserves the original intention of the code better.